components/chat/ScreenDate.tsx

```
@@ -0,0 +1,33 @@
+import { colors } from "@/constants";
+import { StyleSheet, View } from "react-native";
+import CustomText from "../CustomText";
+
+interface ScreenDateProps {}
+
+function ScreenDate({}: ScreenDateProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)